packages/web/src/scss/components/Modal/README.md

```diff
@@ -0,0 +1,54 @@
+# Modal
+
+Modal acts as dialog for any content.
+
+```html
+<button
+  type="button"
+  class="Button Button--primary"
+  data-toggle="modal"
+  data-target="#modal-example-1"
+  aria-controls="modal-example-1"
+  aria-expanded="false"
+>
+  Open Modal Example 1
+</button>
+<div id="modal-example-1" class="Modal">
+  <div class="Modal__dialog">
+    <div class="Modal__content">
+      <div class="Modal__header">
```

Comment:
I'm a little bit confused by the structure. This is what I'd expect:

```html
<div class="Modal">
  <div class="Modal__dialog">
    <div class="Modal__header"></div>
    <div class="Modal__content"></div>
  </div>
</div>
```

It becomes more obvious when I use `<dialog>` element:

```html
<div class="Modal">
  <dialog class="Modal__dialog">
    <div class="Modal__header"></div>
    <div class="Modal__content"></div> <!-- or Modal__body? -->
    <div class="Modal__footer"></div> <!-- future -->
  </dialog>
</div>
```

Could you please investigate if we really need the extra `<div>` around the actual dialog box?